modules/swagger-codegen/src/main/java/io/swagger/codegen/InlineModelResolver.java

```diff
@@ -363,18 +375,45 @@ public Model modelFromProperty(MapProperty object, @SuppressWarnings("unused") S
         String example = null;
 
         Object obj = object.getExample();
-        if(obj != null) {
+        if (obj != null) {
             example = obj.toString();
         }
 
         ArrayModel model = new ArrayModel();
         model.setDescription(description);
         model.setExample(example);
         model.setItems(object.getAdditionalProperties());
+        this.copyVenderExtensions(object, model);
 
         return model;
     }
 
+    /**
+     * Copy vender extensions from Property to Model
+     * 
+     * @param source
+     * @param target
+     */
+    private void copyVenderExtensions(Property source, AbstractModel target) {
+        Map<String, Object> venderExtensions = source.getVendorExtensions();
+        for (String extName : venderExtensions.keySet()) {
+            target.setVendorExtension(extName, venderExtensions.get(extName));
+        }
+    }
```

Comment:
I think in the following example, the codegen cannot know whether the developer want to set the vendor extendsion for property or property's model.
```java
response:
  200:
    schema:
        type: object
        properties:
            ....
        x-my-ext: the-value
```
It may be ok to let the property and the model share the same vendor extensions.

I read my PR again, and I think it make sense to not set the vendor extensions to model.
And, let me fix some bugs in the PR :)